aio/src/styles/2-modules/_contributor.scss

```
@@ -19,6 +19,14 @@ aio-contributor-list {
 .group-buttons {
   margin: 32px auto;
 
+  @media (max-width: 480px) {
+    .a.button.mat-button {
```

```suggestion
    a.button.mat-button {
```

Although, tbh, I would rather use `.filter-button` if possible, since the others are implementation details.

---

we have to at least use .filter-button and .button to rewrite the styles

---

@ajitsinghkaler, is it due to CSS specificity?

---

yes the padding applied in _button .css is applied on the following classes a.button.mat-button

---

I see, thx for clarifying. Let's leave it as is then. Fixing this properly would involve a more thorough rewrite of the styles.